### ISSUE TITLE
[Secure Consul Ph. 2] Update docker-compose for adding consul token header on kong's route

### ISSUE THREAD

**Issue by jim-wang-intel** [edgexfoundry/edgex-compose#71]
After changes made from PR https://github.com/edgexfoundry/edgex-go/pull/3391 merged into master, we need to update the docker compose files in order to make adding consul token header to Kong work.

Depends on issue https://github.com/edgexfoundry/edgex-go/issues/3368
